compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Base.hs

```diff
@@ -230,9 +230,11 @@ data BuiltinExpr
   | BEToTextNumeric              -- :: ∀(s:nat). Numeric s -> Text
   | BEAddNumeric                 -- :: ∀(s:nat). Numeric s -> Numeric s -> Numeric s, crashes on overflow
   | BESubNumeric                 -- :: ∀(s:nat). Numeric s -> Numeric s -> Numeric s, crashes on overflow
-  | BEMulNumeric                 -- :: ∀(s:nat). Numeric s -> Numeric s -> Numeric s, crashes on overflow and underflow, automatically rounds to even (see <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>)
-  | BEDivNumeric                 -- :: ∀(s:nat). Numeric s -> Numeric s -> Numeric s, automatically rounds to even, crashes on divisor = 0 and on overflow
-  | BERoundNumeric              -- :: ∀(s:nat). Int64 -> Numeric s -> Numeric s, the Int64 is the required scale. Note that this doesn't modify the scale of the type itself, it just zeroes things outside that scale out. Can be negative. Crashes if the scale is > 10 or < -27.
+  | BEMulNumeric                 -- :: ∀(s1:nat). ∀(s2:nat). ∀(s3:nat). Numeric s1 -> Numeric s2 -> Numeric s3, crashes on overflow and underflow, automatically rounds to even (see <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>)
```

Comment:
You changed the types here. Do we need to update the spec or is the spec already ahead of this?

Reply:
The comment was out of date, the spec & typechecker were correct here.